pyartcd/pyartcd/pipelines/sigstore_sign.py

```diff
@@ -144,7 +144,7 @@ def _lookup_release_images(self, release_name):
               help="The name of an assembly to be signed. e.g. 4.15.1")
 @click.option("--multi", type=click.Choice(("yes", "no", "only")), default="yes",
               help="Whether to sign multi-arch or arch-specific payloads.")
-@click.option("--sign-release", type=click.Choice(("yes", "no", "only")), default="no",
+@click.option("--sign-release", type=click.Choice(("yes", "no", "only")), default="yes",
```

Comment:
just for the sake of completeness i added a commit to make this the default on the CLI as well (for some reason I didn't have a comment)